README.md

```diff
@@ -1 +1,154 @@
-# rna_folding
+# RNA Folding
+
+In biology and chemistry,
+the properties of a molecule are not solely determined by a set of atoms
+but also by the shape of the molecule. 
+In genetics, the shape of an RNA molecule is largely determined by how it bends back on itself. 
+The sequence of A’s, U’s, G’s, and C’s that make up RNA has 
+certain pairs are drawn together to form hydrogen bonds.
+A sequence of several bonds in a row is called a stem,
+and a stem provides sufficient force to keep the molecule folded together.
+In nature, an RNA molecule will form some stems while avoiding others
+in a manner that  minimizes the free energy of the system. 
+
+This demo program takes am RNA sequence and applies a quadratic model in persuit of the optimal stem configuration.
+
+<p align = "center">
+
+![Figure 1!](readme_imgs/Single_Stem.png "This is a title")
+<p align = "center">
+Fig.1 - An RNA sequence with a single stem of length 4.
+</p>
+
+Predicting stems is important to predicting the properties of the RNA molecule.
+However, prediction is made more complicated by two important factors. 
+
+First, stems are not allowed to overlap. 
+A simple case of overlapping can be illustrated by Figure 1.
+The stem denoted by the pink lines can be denoted with the tuple (2, 5, 14, 17),
+where 2 is the index of the first G,
+4 is the index of the U,
+14 is the index of the A,
+and 17 is the index of the last C.
+This 4-tuple is mapped to a single variable.
+However, the smaller stems (2, 4, 15, 17) and (3, 5, 15, 16) also need to be considered,
+even though the optimal solution will not include them in this case.
+
+Second, the intertwining phenomenon known as psuedoknots are less energetically favorable.
+In Figure 2, we see an example of such a pseudoknot, 
+where one side of a stem occurs in between the two sides of a different stem.
+The use of a quadratic objective allows us to make pseudoknots less likely to occur in optimal solutions,
+increasing overall accuracy.
+
+<p align = "center">
+
+![Figure 2](readme_imgs/pseudoknot2.png)
+<p align = "center">
+Fig.2 - A pseudoknot formed by a stem of length 3 and a stem of length 5.
+</p>
+
+This demo is loosely based on the work of the first paper cited in the References section,
+which is in turn inspired by the second paper in the Reference section.
```

Comment:
Probably a good point to go ahead and switch over to actual citations.  You can check the other examples, but I think we've typically used numbered citations.